client/components/tinymce/plugins/simple-payments/style.scss

```diff
@@ -8,6 +8,7 @@
 		padding: 0;
 		display: flex;
 		flex-direction: column;
+		max-width: none;
```

Comment:
What about also adding a `max-height: none` property here? That would make the modal fullscreen instead of limiting its height to 90%:
<img width="341" alt="modal-mobile-90-height" src="https://user-images.githubusercontent.com/664258/28824767-f0da8630-76c3-11e7-8a35-fb0dd84361ae.png">